migration/migration3.go

```diff
@@ -0,0 +1,53 @@
+package migration
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/query"
+)
+
+type storageJob struct {
+	ID        string
+	APIID     string
+	Cid       cid.Cid
+	CreatedAt int64
+}
+
+// V3StorageJobsIndexMigration contains the logic to upgrade a datastore from
+// version 2 to version 3.
+var V3StorageJobsIndexMigration = Migration{
```

Comment:
Giving this all the migration "3" naming so it doesn't conflict with migration 2 that already exists in `jsign/import`... will make my pending rebase easier.